src/Dropdown.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+// import MyButton from "./MyButton";
+const Dropdown = () => {
+  const [selectedOption, setSelectedOption] = useState({ EmployeeId: "" });
+  const id = ["001", "002", "003", "004", "005"];
+  const deparment = ["frondend", "backend", "QA", "degitalmarketing", "sales"];
+  const Category = ["Hadware", "Software", "Mouse", "keyborad", "headset"];
+
+  const handleOptionChange = (event) => {
+    setSelectedOption(event.target.value);
+    // setSelectedOption(selectedOption);
+  };
+
+  const handleClick = (e) => {
+    alert("success");
+    setSelectedOption(" ");
+  };
+
+  return (
+    <>
+      <div className="container">
+        <div className="profile-color">
+          <h1>Register</h1>
+          <div className="sub-container">
+            <div className="employee-id">
+              <label>Employee ID </label>
+              <select className="dropdown">
+                <option value=""></option>
+                {id.map((id, index) => (
+                  <option key={index} value={id}>
+                    {id}
+                  </option>
+                ))}
+              </select>
+            </div>
+            <div className="department">
+              <label>Department </label>
+              <select className="dropdown">
+                <option value=""></option>
+                {deparment.map((deparment, index) => (
```

Comment:
Change the naming convention of the reference used in the mapping of the department.